### PR TITLE
Adds compat data for DocumentTimeline

### PR DESCRIPTION
**Pull request by bennypowers**

### CODE REVIEW

File: api/DocumentTimeline.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "DocumentTimeline": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentTimeline",
+        "support": {
```
Comment (teoli2003) [inline api/DocumentTimeline.json]: Please order the entries in this order:
"webview_android", "chrome", "chrome_android", "edge", "edge_mobile", "firefox", "firefox_android", "ie", "opera", "opera_android".

Thank you!

File: api/DocumentTimeline.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "DocumentTimeline": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentTimeline/DocumentTimeline",
+          "description": "<code>DocumentTimeline</code> constructor",
+          "support": {
```
Comment (teoli2003) [inline api/DocumentTimeline.json]: Same here.

File: api/DocumentTimeline.json (hunk 1 above)
Comment (bennypowers) [reply api/DocumentTimeline.json]: Order changed. May I suggest that the required order be tested for?

Comment (teoli2003) [reply api/DocumentTimeline.json]: Yes, we should add a test, unfortunately, it is not trivial (read: not possible in the schema), and people usually do the error once then copy/paste another structure, so it does happen only once per contributor ;-) But PRs welcome of course.

**Pull request merged**